Documentation/DataSources/DeprecatedWorkbookRetrievalTool.md

```diff
@@ -0,0 +1,352 @@
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "8d78556d-a4f3-4868-bf06-9e0980246d31",
+            "version": "KqlParameterItem/1.0",
+            "name": "config",
+            "type": 1,
+            "query": "{\"version\":\"ARMEndpoint/1.0\",\"data\":null,\"headers\":[],\"method\":\"GET\",\"path\":\"{resource}/favorites/{favoriteId}\",\"urlParams\":[{\"key\":\"api-version\",\"value\":\"2015-05-01\"},{\"key\":\"sourceType\",\"value\":\"notebook\"},{\"key\":\"canFetchContent\",\"value\":\"true\"}],\"batchDisabled\":false,\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"columns\":[{\"path\":\"$.Config\",\"columnid\":\"Content\"}]}}]}",
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 12
+          }
+        ],
+        "style": "pills",
+        "queryType": 12
+      },
+      "conditionalVisibility": {
+        "parameterName": "debug",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "turn response into param"
+    },
+    {
+      "type": 11,
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "list",
+        "links": [
+          {
+            "id": "fc93ee9e-d5b2-41de-b74a-1fb62f0df49e",
+            "linkTarget": "OpenBlade",
+            "linkLabel": "Open Content as Workbook",
+            "style": "primary",
+            "bladeOpenContext": {
+              "bladeName": "UsageNotebookBlade",
+              "extensionName": "AppInsightsExtension",
+              "bladeParameters": [
+                {
+                  "name": "ComponentId",
+                  "source": "parameter",
+                  "value": "resource"
+                },
+                {
+                  "name": "NewNotebookData",
+                  "source": "parameter",
+                  "value": "config"
+                }
+              ]
+            }
+          }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "config",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "links - 4"
+    }
+  ],
+  "fallbackResourceIds": [
+    "/subscriptions/ad825170-845c-47db-8f00-11978947b089/resourceGroups/IbizaAIExtension/providers/microsoft.insights/components/insightsportal-dev"
+  ],
+  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json"
+}
+
+5. Click Apply at the top right
+6. The grid at the bottom of this workbook will list all the legacy workbooks within the current AppInsights resource.
+7. Click on one of the workbooks in the grid. Your workbook should now look something like this:
+    > ![advanced editor](../Images/DeprecatedWb_RetrievalTool_LegacyWbConversion.png)
+8. Click the button at the bottom of the workbook labeled "Open Content as Workbook"
+9. A new workbook will open with the exact content of the old private workbook that you selected. You can now treat this like any new workbook.
```

Comment:
>A new workbook will open with the exact content of the old private workbook that you selected. You can now treat this like any new workbook. [](http://example.com/codeflow?start=3&length=140)

something like: 

"A new workbook *view* will appear with the content of the old *legacy* workbook that you selected.  In that view you can save this as a standard Workbook."

i wouldn't say "you can treat it like x", i'd tell them what they should do, in this case, save it.

Reply:
i'd do similar clarification of the option 9 of the previous list as well